recipes/kmer-db/meta.yaml

```
@@ -33,3 +36,11 @@ about:
 test:
   commands:
     - kmer-db > /dev/null
+
+extra:
+  additional-platforms:
+    - linux-aarch64
+    - osx-arm64
+  skip-lints:
+    - uses_vcs_url 
+    - missing_hash
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)